website/content/partials/telemetry-metrics/vault/audit/log_request_failure.mdx

```diff
@@ -0,0 +1,14 @@
+### vault.audit.log_request_failure ((#vault-audit-log_request_failure))
+
+Metric type | Value   | Description
+----------- | ------- | -----------
+counter     | number  | Number of audit log request failures across all devices
+
+The number of request failures is a **crucial metric**.
+
+A non-zero value for `vault.audit.log_request_failure` indicates that Vault
+failed to send log data to one of the configured audit log devices. Vault stops all
```

Comment:
This phrasing is a little misleading.  I would say something more like: if all configured audit devices fail to log a request (or response), the request will fail.

Thinking more about it, I think we're actually missing a metric.  You can't tell by looking at this metric whether that happened.  If you have multiple audit devices, one could be failing constantly, but requests still getting processed because the other is fine.  You'd see this metric increasing, but wouldn't be able to distinguish this situation from one where all audit devices are failing at the same time (unless you only have a single device).

We should add a metric that only gets incremented when we fail the request due to all audit devices failing.

Reply:
I tweaked the language a bit based on your comment. Let me know if it's closer to the mark